runtime/src/prioritization_fee_cache.rs

```diff
@@ -543,9 +568,18 @@ mod tests {
             let lock = prioritization_fee_cache.cache.read().unwrap();
```

Comment:
Please update the comment above (and any others that refer to pruning), since that no longer happens.